src/xo-app/dashboard/overview/index.js

```diff
@@ -55,7 +68,131 @@ class PatchesCard extends Component {
   }
 }
 
-// ===================================================================
+@injectIntl
+class ResourceSetCard extends Component {
+  static propTypes = {
+    resourceSet: PropTypes.object.isRequired,
+  }
+
+  _getQuotas = createSelector(
+    () => this.props.resourceSet.limits,
+    limits => {
+      const quotas = {}
+
+      forEach(RESOURCES, resource => {
+        if (limits[resource] != null) {
+          const { available, total } = limits[resource]
+
+          quotas[resource] = {
+            available,
+            total,
+            usage: total - available,
+          }
+        } else {
+          quotas[resource] = {
+            available: 0,
+            total: 0,
+            usage: 0,
```

Comment:
`limits[resource]` being `undefined` should actually mean that there is no limit on this resource.